cli/commands.js

```diff
@@ -103,3 +111,4 @@ Example:
         desc: 'Upgrade the JHipster version, and upgrade the generated application'
     }
 };
+module.exports = Object.assign(defaultCommands, commands);
```

Comment:
Use spread plz
```
module.exports = {
...defaultCommands,
...commands
}
```

Reply:
I'm not used to use this syntax :) 
Thanks for the hint!